lib/config.js

```diff
@@ -340,6 +343,17 @@ config.start = function start() {
                          .pipe(errorLogger || logStream, {end: isPerWorker});
     worker.logFile = logStream;
   }
+
+  function setupExpresUsageForwarding(parentCtl) {
+    var debugExpress = require('./debug')('express-usage');
+    agent().on(
```

Comment:
The idea is that since there won't be any unit-tests, I want to use the debug log to check that the records are emitted/forwarded. Even in the case when there is no parentctl channel.